tox.ini

```diff
@@ -25,6 +25,6 @@ commands = flake8
 basepython = python2.7
 
 [flake8]
-ignore = E251,E128
+ignore = E251,E128,E501
```

Comment:
line length checks should have never been enabled.. so agreed with Martin to disable for now until we can fix them